tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts

```diff
@@ -167,3 +164,63 @@ test('Verify that user can see that Command Helper displays the previous informa
     //Verify Command helper information
     await t.expect(cliPage.cliHelperTitleArgs.textContent).contains(commandForCheck, 'Command Helper information persists after reopening');
 });
+test.only('Verify that user can work with Gears group in Command Helper (RedisGears module)', async t => {
+    const commandForCheck = 'RG.GETEXECUTION';
+    const commandArgumentsCheck = 'RG.GETEXECUTION id [SHARD|CLUSTER]';
+    const readMoreLink = '#rggetexecution';
+    //Open Command Helper
+    await t.click(cliPage.expandCommandHelperButton);
+    //Verify that user can see Gears group in Command Helper (RedisGears module)
+    await cliPage.selectFilterGroupType(COMMAND_GROUP_Gears);
+    //Select one command from the Gears list
+    await t.click(cliPage.cliHelperOutputTitles.withExactText(commandForCheck));
+    //Verify results of opened command
+    await t.expect(cliPage.cliHelperTitleArgs.textContent).eql(commandArgumentsCheck, 'Selected command title');
+    //Verify that user can use Read More link for Gears group in Command Helper (RedisGears module)
+    await t.click(cliPage.readMoreButton);
+    //Check new opened window page with the correct URL
+    await t.expect(getPageUrl()).contains(readMoreLink);
+    //Close the window with external link to switch to the application window
+    await t.closeWindow();
+});
+test('Verify that user can work with Bloom groups in Command Helper (RedisBloom module)', async t => {
+    const commandsForCheck = [
+        'BF.MEXISTS',
+        'CMS.QUERY',
+        'TDIGEST.RESET',
+        'TOPK.LIST',
+        'CF.ADD'
+    ];
+    const commandArgumentsCheck = [
+        'BF.MEXISTS key item [item ...]',
+        'CMS.QUERY key item [item ...]',
+        'TDIGEST.RESET key',
+        'TOPK.LIST key numKeys WITHCOUNT',
+        'CF.ADD key item'
+    ];
+    const externalPage = [
```

Comment:
You could use `let` on top and re-define here.

Reply:
Fixed